llvm/lib/Target/AMDGPU/SIFixSGPRCopies.cpp

```diff
@@ -158,6 +157,20 @@ class SIFixSGPRCopies : public MachineFunctionPass {
   bool tryMoveVGPRConstToSGPR(MachineOperand &MO, Register NewDst,
                               MachineBasicBlock *BlockToInsertTo,
                               MachineBasicBlock::iterator PointToInsertTo);
+};
+
+class SIFixSGPRCopiesLegacy : public MachineFunctionPass {
```

Comment:
I'd have called it SIFixSGPRCopiesLegacy**Pass** but I see there are precedents for this weird naming.

Reply:
If you do that, you end up with the "PassPass" suffix for the initialize* functions